src/Features/SupportTesting/DuskTestable.php

```diff
@@ -120,7 +120,7 @@ static function loadTestComponents()
 
             $components = null;
 
-            try { (new $testClass)->$method(); } catch (\Exception $e) {
+            try { (new $testClass($method))->$method(); } catch (\Exception $e) {
```

Comment:
```suggestion
            try { 
                if (\Orchestra\Testbench\phpunit_version_compare('10.0', '>=') {
                    (new $testClass($method))->$method(); 
                } else {
                    (new $testClass())->$method(); 
                }
            } catch (\Exception $e) {
```

Support both PHPUnit 9 and PHPUnit 10.